product-service/.env

```diff
@@ -5,3 +5,5 @@ DB_PORT=
 DB_NAME=
```

Comment:
Just for interest. Usually `.env` file ignored by `.gitignore`. How do you deploy? Do you hav some other `***.env` file for deployment or do you manually put proper credentials here?